pypesto/visualize/model_fit.py

```diff
@@ -0,0 +1,53 @@
+import petab
+from amici.petab_simulate import simulate_petab
+import amici.petab_import as petab_import
+from petab.visualize import plot_with_vis_spec
+from typing import Union, Sequence
+from ..result import Result
+from amici.petab_objective import rdatas_to_simulation_df
+
+
+def visualize_optimized_model_fit(petab_problem: petab.Problem,
+                                  result: Union[Result, Sequence[Result]],
+                                  **kwargs
+                                  ):
+    """
+    Function calls the petab.visualize file of the petab_problem and
```

Comment:
maybe one-line summary in the first line "Visualize the optimized model fie" or sth like that?

Reply:
how can one call a visualize file